assets/dist/rolodex.js

```diff
@@ -1,5 +1,5 @@
 (function() {
-  angular.module('rolodex', ['templates', 'rolodex.accordion', 'rolodex.alert', 'rolodex.buttons', 'rolodex.datepicker', 'rolodex.dropdown', 'rolodex.modal', 'rolodex.transition']);
```

Comment:
So does removing `templates` get rid of the issue we were having? what was this doing before?